src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -223,6 +223,23 @@ public SqlCommand(string cmdText, SqlConnection connection, SqlTransaction trans
             Transaction = transaction;
         }
 
+        private SqlCommand(SqlCommand from) : this()
+        {
+            CommandText = from.CommandText;
+            CommandTimeout = from.CommandTimeout;
+            CommandType = from.CommandType;
+            Connection = from.Connection;
+            DesignTimeVisible = from.DesignTimeVisible;
+            Transaction = from.Transaction;
+            UpdatedRowSource = from.UpdatedRowSource;
+
+            SqlParameterCollection parameters = Parameters;
+            foreach (object parameter in from.Parameters)
+            {
+                parameters.Add((parameter is ICloneable) ? (parameter as ICloneable).Clone() : parameter);
```

Comment:
I can not get this construct to compile. 
```
Error	CS0103	The name 'c' does not exist in the current context	System.Data.SqlClient	E:\work\gitclone\corefx\src\System.Data.SqlClient\src\System\Data\SqlClient\SqlCommand.cs	239	
```
Is `object as Type newobj` a supported construct? 
I am using VS2015.